DifferentiationInterface/src/utils/printing.jl

```diff
@@ -1,8 +1,10 @@
 function package_name(b::AbstractADType)
     s = string(b)
+    s = chopprefix(s, "ADTypes.")
+    s = chopprefix(s, "Auto")
```

Comment:
a simpler version would be `nameof(typeof(b))[5:(k - 1)]`

Reply:
good point, if I ever run into another similar issue I'll remember it